pipeline-model-definition/pom.xml

```diff
@@ -120,7 +120,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git</artifactId>
-      <scope>test</scope>
+      <optional>true</optional>
```

Comment:
Nope, sadly. The test wouldn't fail with the git plugin just as a test dependency.